ivadomed/scripts/convert_to_onnx.py

```diff
@@ -50,4 +50,5 @@ def main():
 
 
 if __name__ == '__main__':
+    imed_utils.init_ivadomed()
```

Comment:
I performed a test with this script and the version did not show.. would it be possible to put that directly in the main function ?

Reply:
You are very correct! Thanks ++
Corrected, please let me know if the problem remains, cheers